.github/workflows/push-trigger.yml

```
@@ -23,7 +23,37 @@ on:
+          filters: |
+            oidc_ui:
+              - 'oidc-ui/**'
+            esignet_service:
+              - 'esignet-service/**'
+              - 'esignet-service-impl/**'
+              - 'esignet-service-lib/**'
+              - 'pom.xml'
+            api_test:
+              - 'api-test/**'
+            ui_test:
+              - 'ui-test/**'    
```

actionlint (1.7.8)

could not parse as YAML: did not find expected key

(syntax-check)